tools/codechecker_report_hash/tests/projects/macros/macros.plist

```diff
@@ -0,0 +1,172 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Do we need the extra test for a plist file with macros? What is the tested extra feature?